### DISCUSSION TITLE
Are there any performance difference between modifiers?

### DISCUSSION THREAD

**Discussion by Maksimka101** [btwld/mix#473] [Q&A] [answered]
Should I use the `$with.padding` instead of the `$box.padding` or `$with.sizedBox` instead of `$box.height`?

**Comment by leoafarias** [comment, marked as answer]
@Maksimka101 it is the same difference as adding.

`Padding(padding...)` vs `Container(padding: ...)`
or `SizedBox` vs `Container(height:...)`

The `Container` widget adds the Padding widget internally, so it just changes the order of the widgets in the tree. So in the case of $with.padding, or $box.mod.padding, they will behave more like a `margin` than a Container with padding.

Let me know if that makes sense.

**Reply by Maksimka101** [reply]
So 'with' variant is more performant as Padding is a little bit more performant than Container

Why did you added with.padding and with.sizedBox when you already have box alternatives?)

**Reply by leoafarias** [reply]
@Maksimka101, the box is equivalent to the Container. Modifiers are just references to other widgets. There are cases where you want more control over the composability.
